pkg/config/setup/config.go

```diff
@@ -2413,7 +2416,9 @@ func bindEnvAndSetLogsConfigKeys(config pkgconfigmodel.Setup, prefix string) {
 	config.BindEnv(prefix + "dd_url")
 	config.BindEnv(prefix + "additional_endpoints")
 	config.BindEnvAndSetDefault(prefix+"use_compression", true)
-	config.BindEnvAndSetDefault(prefix+"compression_level", 6) // Default level for the gzip/deflate algorithm
+	config.BindEnvAndSetDefault(prefix+"compression_kind", "gzip")
+	config.BindEnvAndSetDefault(prefix+"zstd_compression_level", 1) // Default level for the zstd algorithm
+	config.BindEnvAndSetDefault(prefix+"compression_level", 6)      // Default level for the gzip algorithm
```

Comment:
On main rebase, I kept a `DefaultGzipCompressionLevel` around (which you removed previously?)

Can we consider using the `DefaultZstdCompressionLevel` and `DefaultGzipCompressionLevel` consts here?

Reply:
👍 to keeping constants.